igvm/host.py

```diff
@@ -138,13 +139,13 @@ def put(self, remote_path, local_path, mode='0644'):
             )
 
     def acquire_lock(self, allow_fail=False):
-        if self.dataset_obj['igvm_locked']:
+        if bool(self.dataset_obj['igvm_locked']):
             raise InvalidStateError(
                 'Server "{0}" is already being worked on by another igvm'
                 .format(self.dataset_obj['hostname'])
             )
 
-        self.dataset_obj['igvm_locked'] = True
+        self.dataset_obj['igvm_locked'] = int(time())
```

Comment:
Is this timestamp in users' local timezone? Can we enforce it is in UTC?

It would be way nicer if Serveradmin had a now() function, but I understand this is beyond scope of this ticket.

Reply:
Indeed, I changed it to use UTC timestamp.

Reply:
This gives you utc:
```python3
from datetime import datetime
datetime.utcnow().strftime("%s")
```

Reply:
Ah also adminapi usually want native datatypes, not sure if we ever implemented it for datetime stuff though. We should create an attribute on serveradmin testing and try.

Reply:
So this is a problem. We do have a date datatype and it does take and give proper datetime objects on adminapi side, but we don't have a datetime datatype. We can't save time in it.